frontend/src/lib/components/TaxonomicFilter/TaxonomicFilter.tsx

```diff
@@ -47,16 +53,31 @@ export function TaxonomicFilter({
         window.setTimeout(() => focusInput(), 1)
     }, [])
 
+    const style = {}
+    if (height) {
+        style['height'] = `${height}px`
+    }
+    if (width) {
+        style['width'] = `${width}px`
+    }
```

Comment:
If the unit is pixels, with `width` and `height` you can just pass a number, and `px` will be added automatically. Also using flexible destructuring could make it a bit easier to read... maybe:

```js
const style = {
    ...(width ? { width } : {}),
    ...(height ? { height } : {}),
}
```